c-api/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "picky-c-api"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[lib]
+name = "picky"
+crate-type = ["staticlib", "cdylib"]
+doc = false
+test = false
+doctest = false
+
+[dependencies]
+picky = { version = "6.4.0", default-features = false, features = ["x509", "jose", "http_signature", "time_conversion"], path = "../picky" }
+ffi_helpers = { version = "0.3", git = "https://github.com/CBenoit/ffi_helpers", branch = "error_format_anyhow_debug" }
```

Comment:
Any plan to use the published crate, sync this up or make a Devolutions fork? Maybe add a #TODO at least. Just a thought.

Reply:
Oh, yeah! My patch got merged upstream, but it's not yet published. I'll add a TODO + point directly on the original repo. :+1: